scripts/js/commands/checkPagesRender.ts

```diff
@@ -174,7 +174,6 @@ async function canRender(fp: string): Promise<RenderSuccess | RenderFailure> {
 
 function pathToUrl(path: string): string {
   const strippedPath = path
-    .replace("docs/", "")
```

Comment:
Good change. This is blocked on Arnau's upcoming PR to update the preview app to not strip `/docs`.